interface/src/app/analyze/analyze.js

```diff
@@ -52,6 +53,21 @@ angular.module('adage.analyze', [
         return classList;
       },
 
+      item_tooltip: function(search_item) {
+        // Determine what text to show as a tooltip for this search_item.
+        if (SampleBin.hasItem(search_item)) {
+          return 'already added to analysis';
+        }
+        if (search_item.item_type === 'sample') {
+          return 'add this sample to analysis';
+        }
+        if (search_item.item_type === 'experiment') {
+          return 'add these samples to analysis';
```

Comment:
I really like this feature, of returning a custom message in the tooltip - I think it is very helpful!